tigerpath/majors_and_certificates/majors/POL.json

```diff
@@ -576,7 +678,8 @@
                   "explanation": null,
                   "course_list": [
                     "POL 349",
-                    "POL 352"
+                    "POL 352",

```

Comment:
https://politics.princeton.edu/track-political-economy

(1) Political Economy: either Political Economy (POL 349), Comparative Political Economy for Policy Making (SPI 329/POL 350)*, or Comparative Political Economy (POL 352)*;